### ISSUE TITLE
Picking phase

### ISSUE THREAD

**Issue by SickisM** [ValveSoftware/Dota2-Gameplay#5741]
### Description

See image

### Screenshots

![IMG20221130235435](https://user-images.githubusercontent.com/112749148/204847210-392c64a7-d224-4328-81ce-2db3d12aa664.jpg)

**Comment by Fix-it3** [comment]
Yes im also having this problem.Valve really need to work on their game or ill just play LOL
